python/samples/04.ai.f.dataSource.azureOpenAI/src/bot.py

```diff
@@ -0,0 +1,73 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+
+Description: initialize the app and listen for `message` activitys
+"""
+
+import os
+import sys
+import traceback
+from typing import Optional
+
+from botbuilder.core import MemoryStorage, TurnContext
+from teams import Application, ApplicationOptions, TeamsAdapter
+from teams.ai.models import AzureOpenAIModelOptions, OpenAIModel, OpenAIModelOptions
+from teams.ai.planners import ActionPlanner, ActionPlannerOptions
+from teams.ai.prompts import PromptManager, PromptManagerOptions
+
+from config import Config
+from state import AppTurnState
+
+config = Config()
+
+if config.AZURE_OPENAI_KEY is None:
+    raise RuntimeError(
+        "Missing environment variables - please check that AZURE_OPENAI_KEY is set."
+    )
+
+# Create AI components
+model: OpenAIModel
+
+if config.OPENAI_KEY:
+    model = OpenAIModel(
+        OpenAIModelOptions(api_key=config.OPENAI_KEY, default_model="gpt-3.5-turbo")
+    )
+elif config.AZURE_OPENAI_KEY and config.AZURE_OPENAI_ENDPOINT:
+    model = OpenAIModel(
+        AzureOpenAIModelOptions(
+            api_key=config.AZURE_OPENAI_KEY,
+            default_model="gpt-35-turbo",
+            api_version="2023-03-15-preview",
+            endpoint=config.AZURE_OPENAI_ENDPOINT,
+        )
+    )
+
+prompts = PromptManager(PromptManagerOptions(prompts_folder=f"{os.getcwd()}/src/prompts"))
```

Comment:
we just merged this change yesterday, but please update to

`prompts = PromptManager(PromptManagerOptions(prompts_folder=f"{os.path.dirname(os.path.abspath(__file__))}/prompts"))`

Reply:
updated.